test/test_torch.py

```diff
@@ -2497,6 +2500,7 @@ def test_serialization(self):
         b = [a[i % 2] for i in range(4)]
         b += [a[0].storage()]
         b += [a[0].storage()[1:4]]
+        b + [torch.range(1, 10).int()]
```

Comment:
You seem to discard the result of the addition. I think you wanted to type `+=` here.